contrib/scripts/dumpcerts.sh

```diff
@@ -42,6 +42,17 @@ set -o nounset
 
 USAGE="$(basename "$0") <path to acme> <destination cert directory>"
 
+case `uname` in
+	'Linux')
+		# On Linux, -d should always work. --decode does not work with Alpine's busybox-binary
+		alias decode_base64='base64 -d'
+		;;
+	*)
+		# Max OS-X supports --decode and -D, but --decode may be supported by other platforms as well.
```

Comment:
Just to complete the picture: We could also invert the logic and switch-case on `Darwin` to set `--decode`, and use `-d` in all other cases (Linux, FreeBSD, you name it).

No idea what might be better. I'd go with your current approach when in doubt.